oct/pages/models/checkout_page.py

```diff
@@ -0,0 +1,143 @@
+from pages.base.elements import (
+    RadioButton,
+    Block,
+    InputField,
+    Clickable,
+    CheckBox,
+    Dropdown,
+    Form,
+)
+from oct.pages.base.page import BasePage
+
+
+class CheckoutOptions(Block):
+    contains = {
+        "register_account": {
+            "locator": (
+                "XPATH",
+                '//*[@id="collapse-checkout-option"]/div/div/div[1]/div[1]/label/input',
+            ),
+            "class": RadioButton,
+        },
+        "guest_checkout": {
+            "locator": (
+                "XPATH",
+                '//*[@id="collapse-checkout-option"]/div/div/div[1]/div[2]/label/input',
+            ),
+            "class": RadioButton,
+        },
+        "continue_button": {"locator": ("ID", "button-account"), "class": Clickable},
+    }
+
+
+class ReturningCustomer(Block):
+    contains = {
+        "email": {"locator": ("ID", "input-email"), "class": InputField},
+        "password": {"locator": ("ID", "input-password"), "class": InputField},
+        "login_button": {"locator": ("ID", "button-login"), "class": Clickable},
+    }
+
+    def fill_out(self, **kwargs):
+        self.test.email.fill(kwargs.get("email"))
+        self.test.password.fill(kwargs.get("password"))
+
```

Comment:
ReturnCustomer object does not have a 'test' attribute
You should write fill_out like this
```suggestion
    def fill_out(self, **kwargs):
        self.email.fill(kwargs.get("email"))
        self.password.fill(kwargs.get("password"))
        self.login_button.click()
```